actions/crdjsonschema/openapi2jsonschema.py

```diff
@@ -118,13 +128,27 @@ def write_schema_file(schema, filename):
     f.write(schemaJSON)
     f.close()
     print("{filename}".format(filename=filename))
+    return schema
 
 
 if len(sys.argv) == 0:
     print("missing file")
     exit(1)
 
-for crdFile in sys.argv[1:]:
+
+# This is the object meta v1 schema from Instrumenta
+object_meta_url = "https://raw.githubusercontent.com/instrumenta/kubernetes-json-schema/master/master-standalone/objectmeta-meta-v1.json"
```

Comment:
Please add this file in Git next to the py file. Kubernetes Meta is GA no need to download it every time.

Reply:
Got it. That makes sense.